content/en/docs/concepts/sdk-configuration/sdk-environment-variables.md

```diff
@@ -0,0 +1,142 @@
+# OpenTelemetry Environment Variable Specification
+
+**Status**: [Mixed](document-status.md)
+
+The goal of this specification is to unify the environment variable names between different OpenTelemetry SDK implementations. SDKs MAY choose to allow configuration via the environment variables in this specification, but are not required to. If they do, they SHOULD use the names listed in this document.
+
+## Parsing empty value
+
+**Status**: [Stable](document-status.md)
+
+The SDK MUST interpret an empty value of an environment variable the same way as when the variable is unset.
+
+## Special configuration types
+
+**Status**: [Stable](document-status.md)
+
+### Boolean value
+
+Any value that represents a Boolean MUST be set to true only by the case-insensitive string `"true"`, meaning `"True"` or `"TRUE"` are also accepted, as true.
```

Comment:
Please follow the structure in the other SDK configuration files, ie

Accepted values for Boolean are:
- empty
- unset
- `"true"`, `"True"` or `"TRUE"`: a case insensitive string. **Note** an SDK MUST NOT extend this definition and define additional values interpreted as true.
- `"false"`, `"False"` or `"FALSE"`, empty, unset: any value not expressed as `true` MUST be interpreted as false.

If a value other than the above is used, a warning MUST be logged informing users about the fallback applied.

Reply:
Is there a styling guide that I could refer to?